File: logica-de-programacao/dia-02/10-expressoes-logicas/05-exercicio3-corrigido.cs
// 3. Ano bissexto

using System;

class App
{
    static void Main()
    {
        Console.Write("Digite um ano: ");
        int ano = int.Parse(Console.ReadLine());

        bool divisivelPor4 = ano % 4 == 0;
        bool divisivelPor100 = ano % 100 == 0;
        bool divisivelPor400 = ano % 400 == 0;

        bool ehBissexto = (divisivelPor4 && !divisivelPor100) || divisivelPor400;

        Console.WriteLine($"O ano {ano} é bissexto? {ehBissexto}");
    }
}

File: MM5 Front/curso-adaptacao-js-edicao-2025/dia-01/07-expressoes-logicas/05-exercicio3-corrigido.js
let ano = 2025;

let divisivelPor100 = (ano % 4 === 0) && (ano % 100 !== 0);
let divisivelPor400 = ano % 400 === 0;
let ehBissexto = divisivelPor100 || divisivelPor400;

console.log("O Ano ", ano, " é Bissexto? " + ehBissexto);
